pkgs/by-name/no/nordpass/package.nix

```diff
@@ -0,0 +1,129 @@
+{ fetchurl, lib, stdenv, squashfsTools, xorg, alsa-lib, makeShellWrapper, wrapGAppsHook, openssl, freetype
+, glib, pango, cairo, atk, gdk-pixbuf, gtk3, cups, nspr, nss_latest, libpng, libnotify
+, libgcrypt, systemd, fontconfig, dbus, expat, curlWithGnuTls, zlib, gnome
+, at-spi2-atk, at-spi2-core, libdrm, mesa, libxkbcommon
+, harfbuzz, libsecret, buildFHSEnv
+}:
+
+let
+  version = "5.14.23";
+
+  deps = pkgs: [
+    alsa-lib
+    at-spi2-atk
+    at-spi2-core
+    atk
+    cairo
+    cups
+    curlWithGnuTls
+    dbus
+    expat
+    fontconfig
+    freetype
+    gdk-pixbuf
+    glib
+    gtk3
+    harfbuzz
+    libdrm
+    libgcrypt
+    libnotify
+    libpng
+    libsecret
+    libxkbcommon
+    mesa
+    nspr
+    nss_latest
+    pango
+    stdenv.cc.cc
+    systemd
+    xorg.libICE
+    xorg.libSM
+    xorg.libX11
+    xorg.libxcb
+    xorg.libXcomposite
+    xorg.libXcursor
+    xorg.libXdamage
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXi
+    xorg.libXrandr
+    xorg.libXrender
+    xorg.libXScrnSaver
+    xorg.libxshmfence
+    xorg.libXtst
+    zlib
+  ];
+
+  thisPackage = stdenv.mkDerivation {
+    pname = "nordpass";
+
+    inherit version;
+
+    # determine from
+    # curl -H 'Snap-Device-Series: 16' http://api.snapcraft.io/v2/snaps/info/nordpass
+    src = fetchurl {
+      url = "https://api.snapcraft.io/api/v1/snaps/download/00CQ2MvSr0Ex7zwdGhCYTa0ZLMw3H6hf_178.snap";
```

Comment:
Is this "00CQ2MvSr0Ex7zwdGhCYTa0ZLMw3H6hf_178" reproducible?

Reply:
Not sure by "reproducible". As far as I can tell that is a content based hash. That URL will be the same for this version of the application. Couldn't find a spec on this tho.

Reply:
This is not a content based ID.
This is a unique Id assigned to the "snap". The application ID. This is a fixed Id for the application. The `178` suffix is the version of the app. So, Technically producible as "snap-id" + "_" + "version number"

Reply:
ok. I broke this URL up into components. Seems like the `snapBaseUrl` should go elsewhere but *shrug*